descriptors/ModuleDescriptor-template.json

```diff
@@ -2303,6 +2305,8 @@
       "description" : "Internal permission set for fetching item(s)",
       "subPermissions": [
         "inventory-storage.items.item.get",
+        "circulation-item-storage.items.item.get",
+        "circulation-item-storage.items.collection.get",
```

Comment:
Do we need to add the mod-circulation-item module in requires (optional) dependency section.

Reply:
I added module as optional